### ISSUE TITLE
is_position_locked has no effect on line graphics

### ISSUE THREAD

**Issue by Brow71189** [nion-software/nionswift#248]

**Comment by cmeyer** [comment]
What behavior should is have? For a rectangle, if the position is locked, the rectangle can still be reshaped. Perhaps with the line, the center should remain fixed? And, like the rectangle, if you want to lock is down 100% then `is_position_locked` _and_ `is_shape_locked` should be True.

**Comment by Brow71189** [comment]
I agree, you should set both attributes to lock it down 100%. This does not happen at the moment since only `is_shape_locked` has an effect.
Keeping the center fixed sounds like the right behavior for `is_position_locked`.

**Comment by cmeyer** [comment]
I think that will be nice for adjusting line scans too.
